src/SimpleThings/EntityAudit/Resources/views/Audit/compare.html.twig

```diff
@@ -1,5 +1,21 @@
 {% extends "SimpleThingsEntityAuditBundle::layout.html.twig" %}
 
+{% macro showValue(value) %}
+    {% if value.timestamp is defined %}
+        {{ value|date('m/d/Y') }}
+    {% elseif value is iterable %}
+        <ul>
+        {% for element in value %}
+            <li>{{ element }}</li>
+        {% endfor %}
+        </ul>
+    {% else %}
+        {{ value }}
+    {% endif %}
+{% endmacro %}
+
+{% import _self as helper %}
```

Comment:
@rvanlaak the commit that reintroduced _self in 2.x https://github.com/twigphp/Twig/commit/1015a8f9dba3d5e72d9100b285e77d60defac9dd